### ISSUE TITLE
Improve keyboard navigation

### ISSUE THREAD

**Issue by cbayerlein** [GIScience/ors-map-client#20]
People who cannot use a  pointer device should be able to use the keyboard to use the UI. Therefore it is neccessary to add a tab order and the input focus must be visible. Keyboard shortcuts would also be helpful..

**Comment by amoncaldas** [comment]
Yes, the tab orders are really missing, and we can work on it. Regarding the visibility: is there any input that should be visible and is not?
A list of suggested shortcuts would be welcomed too.

**Comment by cbayerlein** [comment]
I'd give the tab order the highest priority within this issue. Once this is availabe, we can see which input controls need more focus highlighting by tabbing through them. Right now the first tabs give absolutely no visible feedback at all. You can only see it by observing the targeted URL (in the bottom left corner) if you are an experienced user.

Regarding hotkeys/keyboard shortcuts, may first priority would be opening and closing of the input/menu panel, jumping to the input fields for waypoints, the controls underneath that, including the wheelchair parameters - and switching between profiles. Maybe we should create a seperate issue for that

**Comment by amoncaldas** [comment]
Determining manually the tab order of each element is challenging considering that the app is very dynamic and many elements are displayed/hidden depending on interactive conditions. Some other elements are created, removed and reordered on the fly, as a result of the user interactivity. The resolution or window size change also can trigger some elements to be hidden or moved. To implement a comprehensive predefined tab order for each input/element would be a extremely time-consuming task and if a new element is added/moved the tab order of several elements could be affected.

Taking this into consideration, for now it was decided not to define a tab order for each element/input. This will make the tab order to be auto defined based on the order of appearance/location of each element.
Some changes have been introduced to improve the navigation via tab:
- When the accessible mode is activated, the focused element outline is highlighted. 
- Clickable elements that were not defined as inputs are now inputs, so that they are accessible via tab/keyboard
